stubs/resources/views/components/nav-dropdown.blade.php

```diff
@@ -0,0 +1,9 @@
+<li {{ $attributes->merge(['class' => 'dropdown']) }} x-data="{ open: false }" @keyup.escape.window="open = false" @click.away="open = false" @close.stop="open = false">
```

Comment:
`.away` is deprecated and replaced by `.outside`.

see: [Event listener modifier `.away` should be replaced with `.outside`](https://alpinejs.dev/upgrade-guide#away-replace-with-outside)

Reply:
Resolved in [f2654e7](https://github.com/fluid-project/hearth/pull/113/commits/f2654e7434c0bce6d52bcb0a9ea49f284e88466d).